Pipfile

```diff
@@ -31,4 +31,4 @@ docutils = "*"
 flake8 = "*"
 
 [requires]
-python_version = "3.6"
+python_version = "3.7"
```

Comment:
we could specify the latest, but some distributions (I'm looking at _you_ ubuntu LTS) make it stupid difficult to install python 3.7, let alone 3.7.6

Reply:
I'm okay with major/minor only